types/src/ledger_info.rs

```diff
@@ -326,13 +326,13 @@ impl LedgerInfoWithPartialSignatures {
         self.partial_sigs.signatures()
     }
 
-    pub fn aggregate_signatures<T: CryptoHash + Serialize>(
+    pub fn aggregate_signatures(
         &self,
         verifier: &ValidatorVerifier,
-        message: &T,
+        ledger_info: &LedgerInfo,
```

Comment:
it shouldn't take in a ledger_info, it should just use the self.ledger_info

Reply:
Good point - fixed.